### PR TITLE
[KED-1745] Autoformat viz python code using black

### PR DESCRIPTION
**Pull request by mzjp2**
## Description

Make sure `viz` has consistent code formatting standards as the other `kedro` products.

## Development notes

Added `black` step to `make pylint`, `black`ing `package/kedro_viz`, `package/features` and `package/tests`, then ran `make pylint` and let the autoformatting happen. Only edited the code in 3 places, which I've highlighted below

## Checklist

- [ ] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: package/.isort.cfg [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (mzjp2) [inline package/.isort.cfg:1]: removed newline here

File: package/kedro_viz/server.py [hunk 2]
```diff
@@ -286,11 +288,13 @@ def find_child_layers(node_id: str) -> Set[str]:
 def _construct_layer_mapping(catalog):
     if hasattr(catalog, "layers"):  # kedro>=0.16.0
         if catalog.layers is None:
-            return {ds_name: None for ds_name in catalog._data_sets}   # pylint: disable=protected-access
+            return {ds_name: None for ds_name in getattr(catalog, "_data_sets")}
```
Comment (mzjp2) [inline package/kedro_viz/server.py:291]: shortened this line and used `getattr` to be consistent with line 300 below

File: package/tests/test_server.py [hunk 3]
```diff
@@ -615,88 +615,145 @@ def test_format_pipeline_data_no_layers(pipeline, new_catalog_with_layers):
...
+                "node_7": set(),
+            },
+            ["raw", "int", "model_input", "feature"],
+        ),
+        (
+            # disjoint dependency: when two groups of layers have no direct
+            # dependencies,their order is determined by topological order first and
+            # alphabetical order second, which is the default of the toposort library.
+            # In the example below, toposort the layers will give [{c, d}, {b, a}],
+            # so it will become [c, d, a, b] when flattened.
```
Comment (mzjp2) [inline package/tests/test_server.py:741]: broke these comments into one more line to respect line lengths <100 now that `black` indented them

**Comment by richardwestenra** [comment]
@ZainPatelQB Hey I was just preparing a patch release and noticed that this PR is related to #193 which would be included in the release. Should I wait on this PR to be merged before releasing? Is this PR ready to be merged? I see it already has 3 approvals, followed by a bunch more commits.

**Comment by mzjp2** [comment]
> @ZainPatelQB Hey I was just preparing a patch release and noticed that this PR is related to #193 which would be included in the release. Should I wait on this PR to be merged before releasing? Is this PR ready to be merged? I see it already has 3 approvals, followed by a bunch more commits.

Hey Richard, you can go ahead and release without this -- it's got 0 functionality attached to it and is purely developer experience, so doesn't matter _when_ this gets merged. It's currently blocked on legal (Snyk license is failing) and we're waiting on that. Ignore for now :)

**Comment by richardwestenra** [comment]
@ZainPatelQB Great, thanks for the clarification :)

**Comment by yetudada** [comment]
I just got legal approval! Let me update Snyk!

**Pull request merged**